content/docs/demos/local_rate_limit_http.md

```diff
@@ -0,0 +1,342 @@
+---
+title: "Local rate limiting of HTTP requests"
+description: "Configuring local rate limiting for HTTP requests"
+type: docs
+weight: 23
+---
+
+This guide demonstrates how to configure rate limiting for HTTP requests destined to a target host that is a part of an OSM managed service mesh.
+
+## Prerequisites
+
+- Kubernetes cluster running Kubernetes {{< param min_k8s_version >}} or greater.
+- Have OSM installed.
+- Have `kubectl` available to interact with the API server.
+- Have `osm` CLI available for managing the service mesh.
+- OSM version >= v1.2.0.
+
+
+## Demo
+
+The following demo shows a client sending HTTP requests to the `fortio` service. We will see the impact of applying local HTTP rate limiting policies targeting the `fortio` service to control the throughput of requests destined to the service backend.
+
+1. For simplicity, enable [permissive traffic policy mode](/docs/guides/traffic_management/permissive_mode) so that explicit SMI traffic access policies are not required for application connectivity within the mesh.
+    ```bash
+    export osm_namespace=osm-system # Replace osm-system with the namespace where OSM is installed
+    kubectl patch meshconfig osm-mesh-config -n "$osm_namespace" -p '{"spec":{"traffic":{"enablePermissiveTrafficPolicyMode":true}}}'  --type=merge
+    ```
+
+1. Deploy the `fortio` HTTP service in the `demo` namespace after enrolling its namespace to the mesh. The `fortio` HTTP service runs on port `8080`.
+    ```bash
+    # Create the demo namespace
+    kubectl create namespace demo
+
+    # Add the namespace to the mesh
+    osm namespace add demo
+
+    # Deploy fortio TCP echo in the demo namespace
+    kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm-docs/{{< param osm_branch >}}/manifests/samples/fortio/fortio.yaml -n demo
+    ```
+
+    Confirm the `fortio` service pod is up and running.
+
+    ```console
+    $ kubectl get pods -n demo
+    NAME                            READY   STATUS    RESTARTS   AGE
+    fortio-c4bd7857f-7mm6w          2/2     Running   0          22m
+    ```
+
+1. Deploy the `fortio-client` app in the `demo` namespace. We will use this client to send TCP traffic to the `fortio TCP echo` service deployed previously.
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm-docs/{{< param osm_branch >}}/manifests/samples/fortio/fortio-client.yaml -n demo
+    ```
+
+    Confirm the `fortio-client` pod is up and running.
+
+    ```console
+    NAME                            READY   STATUS    RESTARTS   AGE
```

Comment:
To match the one above:
```suggestion
    ```console
    $ kubectl get pods -n demo
    NAME                            READY   STATUS    RESTARTS   AGE
```

Reply:
Done